src/ApiService/IntegrationTests/ToolsTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net;
+using IntegrationTests.Fakes;
+using Microsoft.OneFuzz.Service;
+using Microsoft.OneFuzz.Service.Functions;
+using Xunit;
+using Xunit.Abstractions;
+using Async = System.Threading.Tasks;
+
+namespace IntegrationTests;
+
+[Trait("Category", "Live")]
+public class AzureStorageToolsTest : ToolsTestBase {
+    public AzureStorageToolsTest(ITestOutputHelper output)
+        : base(output, Integration.AzureStorage.FromEnvironment()) { }
+}
+
+public class AzuriteToolsTest : ToolsTestBase {
+    public AzuriteToolsTest(ITestOutputHelper output)
+        : base(output, new Integration.AzuriteStorage()) { }
+}
+
+public abstract class ToolsTestBase : FunctionTestBase {
+    private readonly IStorage _storage;
+
+    public ToolsTestBase(ITestOutputHelper output, IStorage storage)
+        : base(output, storage) {
+        _storage = storage;
+    }
+
+    [Fact]
+    public async Async.Task CanDownload() {
+
+        const int NUMBER_OF_FILES = 20;
+
+        var toolsContainerClient = GetContainerClient(WellKnownContainers.Tools);
+        _ = await toolsContainerClient.CreateIfNotExistsAsync();
+
+        // generate random content 
+        var files = Enumerable.Range(0, NUMBER_OF_FILES).Select((x, i) => (path: i, content: Guid.NewGuid())).ToList();
```

Comment:
use more realistic content for testing ?

Reply:
why ? isn't this enough to test the functionality ?